.gitignore

```diff
@@ -7,3 +7,4 @@
 
 /target
 **/*.rs.bk
+.idea/
```

Comment:
I'm not sure how I feel about editor-specific ignores here.

Reply:
Maybe you are not using the idea editor.

Reply:
Ok, but at least section it of with something like `# Editor specific ignores below`

Reply:
Ok, i submitted